frontend/src/app/_pipes/dashes-to-spaces-and-capitalize.pipe.ts

```diff
@@ -0,0 +1,9 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({ name: 'DashesToSpacesAndCapitalize' })
+export class DashesToSpacesAndCapitalizePipe implements PipeTransform {
```

Comment:
It doesn't like you are actually using this pipe anywhere. Is it necessary?

Reply:
Good catch! I'll remove